aten/src/ATen/native/Scalar.cpp

```diff
@@ -0,0 +1,29 @@
+#include "ATen/ATen.h"
+#include "ATen/NativeFunctions.h"
+
+namespace at {
+namespace native {
+
+Scalar _local_scalar(const Tensor& self) {
```

Comment:
Yeah, let's exclude them. Will add that.

Reply:
is this actually necessary?  I thought about this but decided they wouldn't be exported because we don't have logic to turn a Scalar into a Tensor, so this must be blacklisted somewhere.  But maybe we still have that code lying around from when we had functions/methods that returned Scalars.